cpp/src/io/comp/uncomp.cpp

```diff
@@ -372,7 +392,37 @@ gdf_error io_uncompress_single_h2d(const void *src, gdf_size_type src_size, int
         int zerr = cpu_inflate((uint8_t*)dst.data(), &zdestLen, comp_data, comp_len);
         if (zerr != 0 || zdestLen != uncomp_len)
         {
-    		dst.resize(0);
+            dst.resize(0);
+            return GDF_FILE_ERROR;
+        }
+    }
+    else if (strm_type == IO_UNCOMP_STREAM_TYPE_BZIP2)
+    {
+        size_t src_ofs = 0;
+        size_t dst_ofs = 0;
+        int bz_err = 0;
+    	dst.resize(uncomp_len);
```

Comment:
Missing one space for alignment here

Reply:
I don't seem to see it in the source, it looks like the problem is there are four spaces followed by a tab (copied from line 390 which also includes 4 spaces & 1 tab). Will replace both with spaces.

Reply:
Added a commit to fix indentation